Src/BootCamp.Chapter1/Program.cs

```diff
@@ -1,9 +1,12 @@
-﻿namespace BootCamp.Chapter1
+﻿using System;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5075368)